credentials.json

```diff
@@ -0,0 +1 @@
+{"installed":{"client_id":"283723347574-0vg3qol661e69n5fu95f1ulij5rjlghb.apps.googleusercontent.com","project_id":"quickstart-1583547139681","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://oauth2.googleapis.com/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_secret":"0ojUEYb0w6bTwZyaXVKWzWa0","redirect_uris":["urn:ietf:wg:oauth:2.0:oob","http://localhost"]}}
```

Comment:
this is my credential, feel free to use your own, I just added one window to test so it's just  the last slot